### DISCUSSION TITLE
Which plugins can be passed to transform

### DISCUSSION THREAD

**Discussion by pp-the-pg** [babel/babel#16987] [General]
The documentation isn’t quite clear about which plugins can be passed into the @babel/core transform function. Can I for example pass babel-plugin-transform-react-pug? I wasn’t able to get it to work

**Comment by JLHwung** [comment]
Any Babel plugins are accepted as long as they support Babel 7. Here is [a list of official plugins](https://babel.dev/docs/plugins-list) that works for Babel 7. 

As for `babel-plugin-transform-react-pug`, this is a third-party plugin so you should check the project docs at https://github.com/pugjs/babel-plugin-transform-react-pug. It seems to me that it supports Babel 7 but if not, please open an issue to that project.
